dist/backbone.collectionView.js

```diff
@@ -35,8 +35,8 @@
 		tagName : "ul",
 
 		events : {
-			"mousedown li, td" : "_listItem_onMousedown",
-			"dblclick li, td" : "_listItem_onDoubleClick",
+			"mousedown > li, > tr > td" : "_listItem_onMousedown",
```

Comment:
we can't use `> tr > td` because the `tbody` element is automatically added by some browsers. We will have to just use `tr > td` here.